performance/admin.py

```diff
@@ -0,0 +1,404 @@
+from django.contrib import admin
+from io import StringIO
+from performance import models
+from django_q.tasks import async_task
+
+from frictionless import validate
+
+import os
+import csv
+
+VALID_REPORT_TYPES = ['Provincial Institutions Oversight Performance  Report',
+                      'National Institutions Oversight Performance  Report']
+
+
+def generate_import_report(report_type_validated, frictionless_report, not_matching_departments):
+    report = ""
+    if not report_type_validated:
+        report += "Report preamble must be for one of " + os.linesep
```

Comment:
```suggestion
        report += "Report type must be for one of \n"
```

os.linesep is more fancy than it needs to be. This will match the host OS, while os.linesep is useful if it was handled by the client OS.

Reply:
@emre2038

Reply:
done